hw4_driver.cpp

```diff
@@ -123,7 +123,7 @@ int main()
 */
 
 	std::string * mergeArray = copyArray(unsortedWords);
-	algs.mergeSort(mergeArray, 5000);
+	mergeArray = algs.mergeSort(mergeArray, 5000);
 	printArray(mergeArray);
```

Comment:
Ah yeah, woops I didn't store the string * from mergeSort. printArray was printing the unsorted copy.